PinClustering/PinClustering/Model/Managers/NetworkManager.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  NetworkManager.swift
+//  PinClustering
+//
+//  Created by Norbert Agoston on 06/09/16.
+//  Copyright © 2016 Norbert Agoston. All rights reserved.
+//
+
+import UIKit
+
+class NetworkManager: NSObject {
+    
+    static let sharedInstance = NetworkManager()
+    
+    func httpGet(url: NSURL, callback: (NSArray?, String?) -> Void) {
```

Comment:
don't name the method with GET since in the future you may want to use POST or PUT in your request.